modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/transfer/ProcessImport.java

```diff
@@ -0,0 +1,554 @@
+        reg.addAssociation(successorPath, processAssetPath, ProcessCenterConstants.PREDECESSOR_ASSOCIATION);
+    }
+
+    /**
+     * Add the flow chart of imported process into the registry
+     *
+     * @param processName
+     * @param processVersion
+     * @param processDirPath
+     * @param processAssetPath
+     * @throws UserStoreException
+     * @throws RegistryException
+     * @throws IOException
+     */
+    private void setFlowChart(String processName, String processVersion, String processDirPath, String processAssetPath)
+            throws UserStoreException, RegistryException, IOException {
+        UserRegistry reg = registryService.getGovernanceUserRegistry(user);
+        RegPermissionUtil.setPutPermission(registryService, user, ProcessCenterConstants.AUDIT.PROCESS_FLOW_CHART_PATH);
+
+        Resource flowchartContentResource = reg.newResource();
+
+        Path flowChartJSONFilePath = Paths.get(processDirPath + "/" + ProcessCenterConstants.EXPORTED_FLOW_CHART_FILE);
+        if (Files.exists(flowChartJSONFilePath)) {
+            String flowChartFileContent = "";
+            //Charset charset = Charset.forName(Charsets.US_ASCII"US-ASCII");
+            try (BufferedReader reader = Files.newBufferedReader(flowChartJSONFilePath, Charsets.US_ASCII)) {
+                String line = null;
+                while ((line = reader.readLine()) != null) {
+                    flowChartFileContent += line;
+                }
+            } catch (IOException e) {
+                String errMsg = "Error in reading process's flowChart.json file";
+                throw new IOException(errMsg, e);
+            }
+
+            flowchartContentResource.setContent(flowChartFileContent);
+            flowchartContentResource.setMediaType(MediaType.APPLICATION_JSON);
+            String flowchartContentPath = ProcessCenterConstants.FLOW_CHART + "/" + processName + "/" +
+                    processVersion;
+            reg.put(flowchartContentPath, flowchartContentResource);
+            //add reg association
+            reg.addAssociation(flowchartContentPath, processAssetPath, ProcessCenterConstants.ASSOCIATION_TYPE);
+        }
+    }
+
+    /**
+     * Add the BPMN file of the imported process into the registry
+     *
+     * @param processName
+     * @param processVersion
+     * @param processDirPath
+     * @param processAssetPath
+     * @throws IOException
+     * @throws SAXException
+     * @throws ParserConfigurationException
+     * @throws TransformerException
+     * @throws RegistryException
+     */
+    private void setBPMN(String processName, String processVersion, String processDirPath, String processAssetPath)
+            throws IOException, SAXException, ParserConfigurationException, TransformerException, RegistryException {
+
+        String bpmnFilePathStr = processDirPath + "/" + ProcessCenterConstants.EXPORTED_BPMN_CONTENT_FILE;
```

Comment:
Use **File.separator** instead of "/" every place possible. Because Linux uses forward slash and Windows uses back slash.